src/main/scala/gui/EventsScene.scala

```diff
@@ -0,0 +1,105 @@
+package gui
+
+import java.net.URI
+import java.awt.Desktop
+
+import scalafx.Includes._
+import events.Event
+import scalafx.collections.ObservableBuffer
+import scalafx.scene.Scene
+import scalafx.scene.control.Alert.AlertType
+import scalafx.scene.control.{ Label, TextField}
+import scalafx.stage.Stage
+import scraping.{LiveNationScraper, Monitor, SearchEngine}
+
+class EventsScene(private val sceneWidth: Double, private val sceneHeight: Double, private val stage: Stage
+                  , private val scraper: LiveNationScraper, private val monitor: Monitor) extends Scene(sceneWidth, sceneHeight) {
```

Comment:
This comma is a bit lost :D It would be great if you could do ctrl + alt + l in IntelliJ on all files before commit, it will look better, trust me ;)

Reply:
I did it and IntelliJ didn't find this :(